installer

```diff
@@ -0,0 +1,139 @@
+#!/usr/local/cpython-3.4/bin/python3
+
+"""
+Copy modules from here to --prefix.
+
+Note that we do not scan for directories; instead we have a hardcoded list in all_directories.
+"""
+
+import os
+import shutil
+import sys
+
+
+prefix = os.path.expanduser('~/.micropython/lib')
+directories = []
+all_directories = ['micropython', 'python-ecosys', 'python-stdlib', 'tools', 'unix-ffi']
+
+
+def usage(retval):
```

Comment:
Minor, but imo clearer to pass the  `write` argument directly and have it default to sys.stdout.write: obvious, clearer below in the case stderr gets used beause one doesn't have to wonder what passing `1` means

Reply:
It's like sys.exit(0)